app/src/main/java/com/example/wearit/ui/WardrobeScreen.kt

```diff
@@ -31,7 +35,9 @@ fun WardrobeScreen(
     saveItem: (bitmap: Bitmap) -> Unit,
     getItemPhotoByPhotoFilename: (itemId: String) -> Bitmap,
     setActiveInactive: (item: Item) -> Unit,
-) {
+    goToSettings: () -> Unit,
```

Comment:
A do settingsów nie mieliśmy przechodzić tylko z Pickera?

Reply:
nie pamiętam, dlatego zróbmy to z pickera, usunąłem to z garderoby